python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -495,6 +506,43 @@ def get_schema(cls) -> typing.Dict[str, typing.Any]:
                     for possible debugging scenarios.
                 type: boolean
                 default: false
+              offset_x:
```

Comment:
Oh! these should not be in the `LatissBaseAlign` configuration.

I think having the `additional_configuration` method and having the additional offsets in the other class was better.